velox/connectors/hive/HiveConfig.h

```diff
@@ -49,6 +49,10 @@ class HiveConfig {
   static constexpr const char* kMaxPartitionsPerWritersSession =
       "max_partitions_per_writers";
 
+  /// Maximum number of bucketed count.
+  static constexpr const char* kMaxBucketCount = "max-bucket-count";
+  static constexpr const char* kMaxBucketCountSession = "max-bucket-count";
```

Comment:
session parameters should use underscore `_`

Reply:
@majetideepak Good catch. Fixed.